### DISCUSSION TITLE
what's the name of up-autocomplete step in lsp and down-autocomplete step in lsp?

### DISCUSSION THREAD

**Discussion by imtheaman** [helix-editor/helix#8392] [Q&A] [answered]
I wanna remap key binding for when autocomplete suggestions show up, currently i can only am able to navigate either via arrow keys or tab/S-tab key, i wanna remap it, if ya know the name of those bindings please tell me.

**Comment by the-mikedavis** [comment, marked as answer]
Commands for components like completion currently can't be remapped - this is tracked by https://github.com/helix-editor/helix/issues/5505
